tests/Issues/Issue6927/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\Core\Tests\Issues\Issue6927\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class Fixture
+{
+    /**
+     * @ORM\ManyToOne(targetEntity=Person::class)
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $user;
+
+    public function __construct(object $user, \DateTimeInterface $expiresAt, string $selector, string $hashedToken)
+    {
+        $this->user = $user;
+        $this->initialize($expiresAt, $selector, $hashedToken);
+    }
+}
```

Comment:
Thank you :+1: 

This file should contain the 2nd half of code being separated with `-----`, where you define what you expect.
E.g.

```php
<?php

$value = 10;

?>
-----
<?php

$value = 5;

?>
```

Reply:
I'm not sure what code is should expected. the only I know is the docblock shouldn't removed since it's docblock contain Doctrine thing.

is docblock inside constructor like below is valid?
```
<?php

public function __constructor( /**
     * code block here
     */
     private User $user,
     protected int $counter
) {

}
```

Reply:
is this valid code? this is the test result I get if I didn't use the `$parameters->set(Option::AUTO_IMPORT_NAMES, true);`

```
<?php

namespace Rector\Core\Tests\Issues\Issue6927\Fixture;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Fixture
{
    public function __construct(/**
       * @ORM\ManyToOne(targetEntity=Person::class)
       * @ORM\JoinColumn(nullable=false)
       */
        private object $user,
        \DateTimeInterface $expiresAt,
        string $selector, 
        string $hashedToken
    )
    {
        $this->user = $user;
        $this->initialize($expiresAt, $selector, $hashedToken);
    }
} 
```

Reply:
I'm not sure what code is should expected. the only I know is the docblock shouldn't removed since it's docblock contain Doctrine thing.

is this a valid code? this is the test result I get if I didn't use the `$parameters->set(Option::AUTO_IMPORT_NAMES, true);`

```
<?php

namespace Rector\Core\Tests\Issues\Issue6927\Fixture;

use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Fixture
{
    public function __construct(/**
       * @ORM\ManyToOne(targetEntity=Person::class)
       * @ORM\JoinColumn(nullable=false)
       */
        private object $user,
        \DateTimeInterface $expiresAt,
        string $selector, 
        string $hashedToken
    )
    {
        $this->user = $user;
        $this->initialize($expiresAt, $selector, $hashedToken);
    }
} 
```